### ISSUE TITLE
Pip install fails on Centos 7

### ISSUE THREAD

**Issue by AlexanderNenninger** [qd-cae/qd-cae-python#60]
Hello Devs!

while installing qd on Centos 7, Python 3.7.4 in a conda environment, i get the following errors:
```
(dash_test) ~:()$ pip install qd
Collecting qd
  Using cached https://files.pythonhosted.org/packages/be/2c/1d12319283889fae3b353b66070e919a6ea13559ce31926588bce433df89/qd-0.8.8.tar.gz
Requirement already satisfied: numpy>=1.8 in /opt/anaconda/envs/dash_test/lib/python3.7/site-packages (from qd) (1.16.4)
Requirement already satisfied: diversipy in /opt/anaconda/envs/dash_test/lib/python3.7/site-packages (from qd) (0.8)
Requirement already satisfied: pybind11==2.1.1 in /opt/anaconda/envs/dash_test/lib/python3.7/site-packages (from qd) (2.1.1)
Requirement already satisfied: h5py in /opt/anaconda/envs/dash_test/lib/python3.7/site-packages (from qd) (2.10.0)
Requirement already satisfied: scipy in /opt/anaconda/envs/dash_test/lib/python3.7/site-packages (from diversipy->qd) (1.3.1)
Requirement already satisfied: six in /opt/anaconda/envs/dash_test/lib/python3.7/site-packages (from h5py->qd) (1.12.0)
Building wheels for collected packages: qd
  Building wheel for qd (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /opt/anaconda/envs/dash_test/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-690dtvp1/qd/setup.py'"'"'; __file__='"'"'/tmp/pip-install-690dtvp1/qd/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-gqc_pnes --python-tag cp37
       cwd: /tmp/pip-install-690dtvp1/qd/
  Complete output (54 lines):
  FEMZIP library not found. Compiling without FEMZIP.
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-3.7
  creating build/lib.linux-x86_64-3.7/qd
  copying qd/__init__.py -> build/lib.linux-x86_64-3.7/qd
  creating build/lib.linux-x86_64-3.7/qd/cae
  copying qd/cae/ArrayD3plot.py -> build/lib.linux-x86_64-3.7/qd/cae
  copying qd/cae/Binout.py -> build/lib.linux-x86_64-3.7/qd/cae
  copying qd/cae/D3plot.py -> build/lib.linux-x86_64-3.7/qd/cae
  copying qd/cae/dyna.py -> build/lib.linux-x86_64-3.7/qd/cae
  copying qd/cae/KeyFile.py -> build/lib.linux-x86_64-3.7/qd/cae
  copying qd/cae/lsda_py3.py -> build/lib.linux-x86_64-3.7/qd/cae
  copying qd/cae/Part.py -> build/lib.linux-x86_64-3.7/qd/cae
  copying qd/cae/RawD3plot.py -> build/lib.linux-x86_64-3.7/qd/cae
  copying qd/cae/_dyna_utils.py -> build/lib.linux-x86_64-3.7/qd/cae
  copying qd/cae/__init__.py -> build/lib.linux-x86_64-3.7/qd/cae
  creating build/lib.linux-x86_64-3.7/qd/cae/beta
  copying qd/cae/beta/MetaCommunicator.py -> build/lib.linux-x86_64-3.7/qd/cae/beta
  copying qd/cae/beta/__init__.py -> build/lib.linux-x86_64-3.7/qd/cae/beta
  creating build/lib.linux-x86_64-3.7/qd/cae/resources
  copying qd/cae/resources/__init__.py -> build/lib.linux-x86_64-3.7/qd/cae/resources
  creating build/lib.linux-x86_64-3.7/qd/numerics
  copying qd/numerics/sampling.py -> build/lib.linux-x86_64-3.7/qd/numerics
  copying qd/numerics/__init__.py -> build/lib.linux-x86_64-3.7/qd/numerics
  copying qd/cae/beta/meta_remote_control -> build/lib.linux-x86_64-3.7/qd/cae/beta
  copying qd/cae/beta/meta_remote_control.exe -> build/lib.linux-x86_64-3.7/qd/cae/beta
  copying qd/cae/beta/msvcr71.dll -> build/lib.linux-x86_64-3.7/qd/cae/beta
  copying qd/cae/resources/chroma.min.js -> build/lib.linux-x86_64-3.7/qd/cae/resources
  copying qd/cae/resources/jquery.min.js -> build/lib.linux-x86_64-3.7/qd/cae/resources
  copying qd/cae/resources/jszip.min.js -> build/lib.linux-x86_64-3.7/qd/cae/resources
  copying qd/cae/resources/three.min.js -> build/lib.linux-x86_64-3.7/qd/cae/resources
  copying qd/cae/resources/html.template -> build/lib.linux-x86_64-3.7/qd/cae/resources
  running build_ext
  building 'dyna_cpp' extension
  creating build/temp.linux-x86_64-3.7
  creating build/temp.linux-x86_64-3.7/qd
  creating build/temp.linux-x86_64-3.7/qd/cae
  creating build/temp.linux-x86_64-3.7/qd/cae/dyna_cpp
  creating build/temp.linux-x86_64-3.7/qd/cae/dyna_cpp/python_api
  creating build/temp.linux-x86_64-3.7/qd/cae/dyna_cpp/db
  creating build/temp.linux-x86_64-3.7/qd/cae/dyna_cpp/dyna
  creating build/temp.linux-x86_64-3.7/qd/cae/dyna_cpp/dyna/d3plot
  creating build/temp.linux-x86_64-3.7/qd/cae/dyna_cpp/dyna/keyfile
  creating build/temp.linux-x86_64-3.7/qd/cae/dyna_cpp/utility
  gcc -pthread -B /opt/anaconda/envs/dash_test/compiler_compat -Wl,--sysroot=/ -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Iqd/cae -I/opt/anaconda/envs/dash_test/lib/python3.7/site-packages/numpy/core/include -Iinclude -I/opt/anaconda/envs/dash_test/include/python3.7m -c qd/cae/dyna_cpp/python_api/pybind_wrapper.cpp -o build/temp.linux-x86_64-3.7/qd/cae/dyna_cpp/python_api/pybind_wrapper.o -std=c++11 -fPIC -D_GLIBCXX_USE_CXX11_ABI=0 -DQD_VERSION="0.8.8" -O3 -Wno-unknown-pragmas
  cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++ [enabled by default]
  qd/cae/dyna_cpp/python_api/pybind_wrapper.cpp:41:46: fatal error: dyna_cpp/python_api/docstrings.cpp: No such file or directory
   #include <dyna_cpp/python_api/docstrings.cpp>
                                                ^
  compilation terminated.
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for qd
  Running setup.py clean for qd
Failed to build qd
Installing collected packages: qd
  Running setup.py install for qd ... error
    ERROR: Command errored out with exit status 1:
     command: /opt/anaconda/envs/dash_test/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-690dtvp1/qd/setup.py'"'"'; __file__='"'"'/tmp/pip-install-690dtvp1/qd/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-fx6uoj3u/install-record.txt --single-version-externally-managed --compile
         cwd: /tmp/pip-install-690dtvp1/qd/
    Complete output (54 lines):
    FEMZIP library not found. Compiling without FEMZIP.
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.7
    creating build/lib.linux-x86_64-3.7/qd
    copying qd/__init__.py -> build/lib.linux-x86_64-3.7/qd
    creating build/lib.linux-x86_64-3.7/qd/cae
    copying qd/cae/ArrayD3plot.py -> build/lib.linux-x86_64-3.7/qd/cae
    copying qd/cae/Binout.py -> build/lib.linux-x86_64-3.7/qd/cae
    copying qd/cae/D3plot.py -> build/lib.linux-x86_64-3.7/qd/cae
    copying qd/cae/dyna.py -> build/lib.linux-x86_64-3.7/qd/cae
    copying qd/cae/KeyFile.py -> build/lib.linux-x86_64-3.7/qd/cae
    copying qd/cae/lsda_py3.py -> build/lib.linux-x86_64-3.7/qd/cae
    copying qd/cae/Part.py -> build/lib.linux-x86_64-3.7/qd/cae
    copying qd/cae/RawD3plot.py -> build/lib.linux-x86_64-3.7/qd/cae
    copying qd/cae/_dyna_utils.py -> build/lib.linux-x86_64-3.7/qd/cae
    copying qd/cae/__init__.py -> build/lib.linux-x86_64-3.7/qd/cae
    creating build/lib.linux-x86_64-3.7/qd/cae/beta
    copying qd/cae/beta/MetaCommunicator.py -> build/lib.linux-x86_64-3.7/qd/cae/beta
    copying qd/cae/beta/__init__.py -> build/lib.linux-x86_64-3.7/qd/cae/beta
    creating build/lib.linux-x86_64-3.7/qd/cae/resources
    copying qd/cae/resources/__init__.py -> build/lib.linux-x86_64-3.7/qd/cae/resources
    creating build/lib.linux-x86_64-3.7/qd/numerics
    copying qd/numerics/sampling.py -> build/lib.linux-x86_64-3.7/qd/numerics
    copying qd/numerics/__init__.py -> build/lib.linux-x86_64-3.7/qd/numerics
    copying qd/cae/beta/meta_remote_control -> build/lib.linux-x86_64-3.7/qd/cae/beta
    copying qd/cae/beta/meta_remote_control.exe -> build/lib.linux-x86_64-3.7/qd/cae/beta
    copying qd/cae/beta/msvcr71.dll -> build/lib.linux-x86_64-3.7/qd/cae/beta
    copying qd/cae/resources/chroma.min.js -> build/lib.linux-x86_64-3.7/qd/cae/resources
    copying qd/cae/resources/jquery.min.js -> build/lib.linux-x86_64-3.7/qd/cae/resources
    copying qd/cae/resources/jszip.min.js -> build/lib.linux-x86_64-3.7/qd/cae/resources
    copying qd/cae/resources/three.min.js -> build/lib.linux-x86_64-3.7/qd/cae/resources
    copying qd/cae/resources/html.template -> build/lib.linux-x86_64-3.7/qd/cae/resources
    running build_ext
    building 'dyna_cpp' extension
    creating build/temp.linux-x86_64-3.7
    creating build/temp.linux-x86_64-3.7/qd
    creating build/temp.linux-x86_64-3.7/qd/cae
    creating build/temp.linux-x86_64-3.7/qd/cae/dyna_cpp
    creating build/temp.linux-x86_64-3.7/qd/cae/dyna_cpp/python_api
    creating build/temp.linux-x86_64-3.7/qd/cae/dyna_cpp/db
    creating build/temp.linux-x86_64-3.7/qd/cae/dyna_cpp/dyna
    creating build/temp.linux-x86_64-3.7/qd/cae/dyna_cpp/dyna/d3plot
    creating build/temp.linux-x86_64-3.7/qd/cae/dyna_cpp/dyna/keyfile
    creating build/temp.linux-x86_64-3.7/qd/cae/dyna_cpp/utility
    gcc -pthread -B /opt/anaconda/envs/dash_test/compiler_compat -Wl,--sysroot=/ -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Iqd/cae -I/opt/anaconda/envs/dash_test/lib/python3.7/site-packages/numpy/core/include -Iinclude -I/opt/anaconda/envs/dash_test/include/python3.7m -c qd/cae/dyna_cpp/python_api/pybind_wrapper.cpp -o build/temp.linux-x86_64-3.7/qd/cae/dyna_cpp/python_api/pybind_wrapper.o -std=c++11 -fPIC -D_GLIBCXX_USE_CXX11_ABI=0 -DQD_VERSION="0.8.8" -O3 -Wno-unknown-pragmas
    cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++ [enabled by default]
    qd/cae/dyna_cpp/python_api/pybind_wrapper.cpp:41:46: fatal error: dyna_cpp/python_api/docstrings.cpp: No such file or directory
     #include <dyna_cpp/python_api/docstrings.cpp>
                                                  ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    ----------------------------------------
ERROR: Command errored out with exit status 1: /opt/anaconda/envs/dash_test/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-690dtvp1/qd/setup.py'"'"'; __file__='"'"'/tmp/pip-install-690dtvp1/qd/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-fx6uoj3u/install-record.txt --single-version-externally-managed --compile Check the logs for full command output.

```

**Comment by lasso-codie** [comment]
Seems a header file is missing in the release, thanks for the report. 

A quick fix is to clone the git repo itself as explained in the [Compilation Guide](https://qd-cae.github.io/qd-cae-python/build/html/compilation_guide.html#compilation-on-linux):

```bash
git clone --depth 1 --branch master --single-branch https://github.com/qd-cae/qd-cae-python.git
cd qd-cae-python
sudo python setup.py install
```

**Comment by lasso-codie** [comment]
Will be fixed in 0.8.9
